unified_planning/model/timing.py

```diff
@@ -28,28 +28,49 @@ class TimepointKind(Enum):
 
 
 class Timepoint:
-    def __init__(self, kind: TimepointKind):
+    def __init__(self, kind: TimepointKind, container=None):
+        """Creates a new timepoint.
+
+        Parameters
+        ----------
+        kind: TimepointKind
+          Kind of the timepoint.
+        container: Optional[Subtask, ActionInstance]
+          Container in which the timepoint is defined.
+          If not set, then a start/end timepoint refers to the enclosing action or method.
+          The container must have an `identifier` property that is a string uniquely identifying it.
```

Comment:
`ActionInstance` does not have an `identifier` property

Reply:
Right, I removed the reference to `ActionInstance`. 
Essentially, this is designed to eventually support having an `ActionInstance` there which I believe will be useful for STN/POP plans.
However, this is not required by this PR or hierarchical planning in general so I removed it for now.